content/1_General/Input_Output.mdx

```diff
@@ -790,34 +839,34 @@ A different approach to file I/O in Python is to still use the `open()` method,
 but use the built-in functions `.readline()` or `.readlines()`:
 	
 ```py
-# Note: The second argument can be omitted in the open()
-      #	command for read-only files
+"""
+Note: The second argument can be omitted in the open()
+command for read-only files
+"""
 fin = open("problemname.in", "r") 
 fout = open("problemname.out", "w")
 
 # One way to read the file using .readline()
-line1 = fin.readline() 
+line1 = fin.readline()
+# readline() will pick up where you left off 
 line2 = fin.readline()
-line3 = fin.readline() # The .readline() method will pick up where you left off
+line3 = fin.readline()
 
-# OR:
-# Use a for loop with the .readlines() method
-
-linelist = []
+# Another way is to use a for loop and .readlines()
+line_list = []
 for line in fin.readlines():
-	# Separate the values, etc here
+	pass  # Do what you need to do here
 
-# printing linelist would give [line1, line2, line3]			
+# printing line_list would give [line1, line2, line3]			
 
 # Output:
-fout.write(output_text) # Write to the output file
+fout.write(output_text)  # Write to the output file
```

Comment:
did you mean to put two spaces here? if so, add space for the other comment too

Reply:
thx ryan

Reply:
wait what i don't see it in my file

Reply:
oh wait nvm, you deleted the other one. im blind